third_party/2and3/redis/client.pyi

```diff
@@ -81,6 +81,28 @@ class Redis(object):
     def pubsub(self, shard_hint: Any = ..., ignore_subscribe_messages: bool = ...) -> PubSub: ...
     def execute_command(self, *args, **options): ...
     def parse_response(self, connection, command_name, **options): ...
+    def acl_cat(self, category: Optional[Text]) -> List[str]: ...
+    def acl_deluser(self, username: Text) -> int: ...
+    def acl_genpass(self) -> Text: ...
+    def acl_getuser(self, username: Text) -> Union[None, Dict[str, Union[List[str], bool]]]: ...
+    def acl_list(self) -> List[Text]: ...
+    def acl_load(self) -> bool: ...
+    def acl_setuser(
+        self,
+        username: Text,
+        enabled: bool,
+        nopass: bool,
+        passwords: bool,
+        hashed_passwords: Optional[List[Text]],
+        categories: Optional[List[Text]],
+        commands: Optional[List[Text]],
+        keys: Optional[List[Text]],
+        reset: bool,
+        reset_keys: bool,
+        reset_passwords: bool,
```

Comment:
All of these options have default values, so they should have `= ...`.

A few more from looking at the code:

* `hashed_passwords` can use the looser `Sequence` type and also accepts plain strings, so I suggest using `Optional[Union[Text, Sequence[Text]]]`.
* Similar for `passwords`.
* `categories`, `commands`, and `keys` also can use `Sequence` instead of `List.

Reply:
Updated with `...` for default values and Sequence for List